SEDaily-IOS.xcodeproj/project.pbxproj

```diff
@@ -129,7 +138,6 @@
 		16FA84021F8D323700A45D9B /* SkeletonCollectionView.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = SkeletonCollectionView.swift; sourceTree = "<group>"; };
 		1E286DE21FE4C5BA00644C1D /* TestHookEventTableViewCell.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = TestHookEventTableViewCell.swift; sourceTree = "<group>"; };
 		1E286DE41FE4C63200644C1D /* TestHookManager.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = TestHookManager.swift; sourceTree = "<group>"; };
-		1E286DE61FE4C64800644C1D /* TestHookBool.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = TestHookBool.swift; sourceTree = "<group>"; };
```

Comment:
I'm getting an error for this: `<unknown>:0: error: no such file or directory: '/Users/keithholliday/Desktop/TestHookBool.swift'`

Reply:
I think I saw this before but forgot how I resolved it.  Did you try doing a clean build? If so I'll have to take a look, might be an issue with this PR.

Also note that this PR is going to need to be updated since the API for comments has changed in preparation for the forum (so it won't work unless you have it pointed to a local branch with older code).

Reply:
Yea, I cleaned and removed generated files but with no luck. And okay. I think copying the comment code to a new PR and updating it would be best since there is a lot of extra code and commits

Reply:
@jasonify @TheHollidayInn I hit this error. Just remove the file from XCode and re-add it again. It should fix it.